docs/buildrun.md

```diff
@@ -159,6 +160,26 @@ spec:
   state: "BuildRunCanceled"
 ```
 
+## Automatic `BuildRun` deletion
+
+We have two controllers that ensure that buildruns can be deleted automatically if required. This is ensured by adding `retention` parameters in build specifications.
+
+- TTL parameters: These are used by the buildrun_ttl_cleanup controller.
+  - `build.spec.retention.ttlAfterFailed`: The buildrun is deleted if the mentioned duration of time has passed and the buildrun has failed.
+  - `build.spec.retention.ttlAfterSucceeded`: The buildrun is deleted if the mentioned duration of time has passed and the buildrun has succeeded.
+- Limit parameters: These are used by the build_limit_cleanup controller.
+  - `build.spec.retention.failedLimit`: This parameter ensures that the number of failed buildruns do not exceed this limit. If the limit is exceeded, the oldest failed buildruns are deleted till the limit is satisfied.
+  - `build.spec.retention.succeededLimit`: This parameter ensures that the number of successful buildruns do not exceed this limit. If the limit is exceeded, the oldest successful buildruns are deleted till the limit is satisfied.
+
+### Buildrun ttl cleanup controller
```

Comment:
same as the other comment on the `docs/build.md` section regarding controller docs.